### ISSUE TITLE
dirty files in the git repo

### ISSUE THREAD

**Issue by devoncarew** [flutter/flutter#9460]
I see this now in the flutter checkout:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	examples/flutter_gallery/android/app/src/main/java/
	examples/flutter_gallery/ios/Runner/PluginRegistry.h
	examples/flutter_gallery/ios/Runner/PluginRegistry.m
```

We should commit these files or update the .gitignore, as appropriate. /cc'ing @jakobr-google who probably has the most context here.

**Comment by eseidelGoogle** [comment]
There are several related issues on file. :)  @mit-mit

**Comment by mit-mit** [comment]
From further discussion with Jakob, the fix in #9555 to exclude them from git has the issue that you can not git clone and then open in Xcode and build (as these files are needed to build).

A better fix would be to actually commit these to source control, incl. the reference to them from runner.xcodeproj (iOS), and AppDelegate & Activity (Android).

**Comment by mravn-google** [comment]
`GeneratedPluginRegistrant` files are both checked into the Flutter repo and mentioned in `.gitignore`. So no dirty files in Flutter repo after running examples.
